src/Identity/test/Identity.Test/PasswordValidatorTest.cs

```diff
@@ -24,15 +24,15 @@ public async Task ValidateThrowsWithNullTest()
 
         // Act
         // Assert
-        await Assert.ThrowsAsync<ArgumentNullException>("password", () => validator.ValidateAsync(null, null, null));
         await Assert.ThrowsAsync<ArgumentNullException>("manager", () => validator.ValidateAsync(null, null, "foo"));
     }
 
     [Theory]
-    [InlineData("")]
-    [InlineData("abc")]
-    [InlineData("abcde")]
-    public async Task FailsIfTooShortTests(string input)
+    [InlineData(null)]  
+    [InlineData("")]    
+    [InlineData("abc")] 
+    [InlineData("abcde")] 
```

Comment:
We should remove the trailing whitespace.

```suggestion
    [InlineData(null)]
    [InlineData("")]
    [InlineData("abc")]
    [InlineData("abcde")]
```